console/console-init/ui/src/modules/iot-device/components/ReviewDevice/MetaDataReviewRow.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React, { useState } from "react";
+import { StyleSheet, css } from "aphrodite";
+import { AngleRightIcon, AngleDownIcon } from "@patternfly/react-icons";
+import { GridItem, TextInput, Grid, Button } from "@patternfly/react-core";
+import { MetaDataReviewRows } from "./MetaDataReviewRows";
+
+const styles = StyleSheet.create({
```

Comment:
Can you please try to use `rem` for css. It will help later when we will remove aphrodite lib.